ui/app/components/clients/page/overview.hbs

```diff
@@ -0,0 +1,30 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+{{#if this.byMonthActivityData}}
+  <Clients::RunningTotal
+    @selectedAuthMethod={{this.authMount}}
```

Comment:
I noticed below that this arg was renamed `@authMount={{this.selectedAuthMethod}}` it'd be nice as a follow-on cleanup task to have these args be named consistently if possible. What do you think?

Also, this is available on the controller but passed in as an arg - should this be updated to `@authMount` so it's clear it's an arg to this component? `this.` made me look for a getter or tracked variable in the component class

Reply:
Yes I think that's a good idea! I started a cleanup ticket and added a todo for auditing the old component args. You're right about `this.authMount` good catch! After looking into it further it looks like that property isn't used anymore so I will remove the arg entirely.